scripts/provisioning-tests

```diff
@@ -39,7 +39,9 @@ eval "$(grep '^ENV CATTLE_WINS_AGENT' package/Dockerfile | awk '{print "export "
 eval "$(grep '^ENV CATTLE_CSI_PROXY_AGENT' package/Dockerfile | awk '{print "export " $2 "=" $3}')"
 eval "$(grep '^ENV CATTLE_KDM_BRANCH' package/Dockerfile | awk '{print "export " $2 "=" $3}')"
 
-export SOME_K8S_VERSION=$(curl https://raw.githubusercontent.com/rancher/kontainer-driver-metadata/dev-v2.7/data/data.json | jq -r ".$DIST.channels[0].latest")
+# Get the last release for $DIST, which is usually the latest version or an experimental version.
+export SOME_K8S_VERSION=$(curl -sS https://raw.githubusercontent.com/rancher/kontainer-driver-metadata/dev-v2.7/data/data.json | jq -r ".$DIST.releases[-1].version")
+echo "K8s version: $SOME_K8S_VERSION"
```

Comment:
@sbstp , I'm late to the part here, but this `echo` looks redundant and possibly misleading as we're very vague about what this k8s version is for.
The very next (existing) `echo` is a lot more specific and I think is sufficient. So it looks like leftover debugging code.
cc: @HarrisonWAffel @jiaqiluo